netutils/bandwidth.py

```diff
@@ -123,7 +124,7 @@ def bits_to_name(  # pylint: disable=too-many-branches,too-many-return-statement
     for bit_type, val in BITS_MAPPING.items():
         if val["low"] <= speed < val["high"]:
             if nbr_decimal == 0:
-                nbr_decimal = None
+                nbr_decimal = None  # type: ignore
```

Comment:
I believe that `round(x, None) == round(x)` for all `x`, right? Meaning we could drop this branch and the corresponding `# type: ignore`.